src/Microsoft.PowerShell.Commands.Utility/commands/utility/MarkdownOptionCommands.cs

```diff
@@ -256,7 +270,43 @@ public class GetMarkdownOptionCommand : PSCmdlet
         /// </summary>
         protected override void EndProcessing()
         {
-            WriteObject(this.CommandInfo.Module.SessionState.PSVariable.GetValue(MarkdownOptionInfoVariableName, new PSMarkdownOptionInfo()));
+            var option = this.CommandInfo.Module == null
+                // If we don't have a moduleInfo, like in PowerShell hosting scenarios, use a concurrent dictionary.
+                ? PSMarkdownOptionInfoCache.Get(Runspace.DefaultRunspace.InstanceId)
+                // If we have the moduleInfo then store are module scope variable
```

Comment:
I guess CodeFactor will reports style issues for the comments.
We could move both to line  273.

Reply:
Fixed.